docs/user/aws/install_upi.md

```diff
@@ -18,6 +18,18 @@ $ openshift-install create install-config
 ? Pull Secret [? for help]
 ```
 
+### Optional: Create Encrypted AMIs
+
+The IPI-based installer creates an encrypted AMI by default. If you wish to have an encrypted AMI for UPI-based
+installs, you will need to create it directly. You can find a list of the appropriate base AMIs 
```

Comment:
we technically don't have to point them to rhcos.json right? telling them take the AMI you would have used (before this), encrypt it and use that new one... ?

Reply:
We actually never helped them find this. In the pubs, they point to and keep a table updated. They should reference that when they add this to the pubs. Here in the repo, we provide no help/guidance. Linking to the file here in the repo seems fair/right and then it's changed to a pointer to the table when included in the doc.